tests/unit/s2n_tls13_handshake_state_machine_test.c

```diff
@@ -493,13 +603,11 @@ int main(int argc, char **argv)
                 break;
             }
 
-            if (IS_NEGOTIATED(handshake_type)) {
-                EXPECT_TRUE(change_cipher_spec_found);
-            }
+            EXPECT_TRUE(change_cipher_spec_found);
         }
     }
 
-    /* Test: TLS1.3 s2n_conn_set_handshake_type sets FULL_HANDSHAKE, HELLO_RETRY_REQUEST, and CLIENT_AUTH*/
+    /* Test: TLS1.3 s2n_conn_set_handshake_type sets only handshake flags allowed by TLS1.3 */
```

Comment:
For future readability: could we update this comment to explicitly call out the flags allowed by TLS1.3? Thanks.

Reply:
I intentionally removed the flag names from comments in these tests because some were already out of date and had not been updated as the tests were updated. Putting the list in a comment is repetitive and leads to the potential for the comment and the code to disagree. I think the code can speak for itself here-- the lists of flags are pretty clear.

Reply:
The reasoning is fair - to maintain a single point of truth, but I find that it requires quite a bit of analyzing the code to obtain the list. For the tls1.2 case the EXPECT_TRUEs list of the flags are together and easy to read, but for TLS1.2 the list is not together and for a quick look up on tls1.3 flags one has to read more code.